scripts/src/java/org/oppia/android/scripts/label/AccessibilityLabelCheck.kt

```diff
@@ -5,126 +5,145 @@ import org.w3c.dom.Node
 import org.w3c.dom.NodeList
 import java.io.File
 import java.io.FileInputStream
-import java.util.stream.Collectors
-import java.util.stream.IntStream
 import javax.xml.parsers.DocumentBuilderFactory
 
 /**
  * Script for ensuring that all the Activities in the codebase are defined with accessibility
  * labels.
  *
  * Usage:
- *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_app_level
- *   _manifest_file>
+ *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_manifest
```

Comment:
```suggestion
 *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <paths_to_manifest
```

However, it should be something like:

``[paths to manifest files...]`` to indicate that it's space-separated.

Reply:
Done.